src/client/components/filter-menu/time-filter-menu/time-filter-menu.tsx

```diff
@@ -77,12 +72,12 @@ export class TimeFilterMenu extends React.Component<TimeFilterMenuProps, TimeFil
     if (!dimension) return null;
     const menuSize = Stage.fromSize(MENU_WIDTH, 410);
 
-    const tabs = [TimeFilterMenu.RELATIVE_TAB, TimeFilterMenu.FIXED_TAB].map(name => {
+    const tabs = [TimeFilterTab.RELATIVE, TimeFilterTab.FIXED].map(name => {
       return {
         isSelected: tab === name,
-        title: (name === TimeFilterMenu.RELATIVE_TAB ? STRINGS.relative : STRINGS.fixed),
+        title: (name === TimeFilterTab.RELATIVE ? STRINGS.relative : STRINGS.fixed),
```

Comment:
I'd name it in the current scope or extract as a function. `title: getTitle(name)` - how sweet it looks, doesn't it?